File: NativeWrappers/Windows/SimplySaveWindows/SimplySaveWindows/DataTransfer/NoteData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace SimplySaveWindows.DataTransfer
{
    public class NoteData
    {
        [JsonPropertyName("id")]
        public string Id { get; set; }
        [JsonPropertyName("title")]
        public string Title { get; set; }
        [JsonPropertyName("folderId")]
        public string FolderId { get; set; }
        [JsonPropertyName("createdTimestamp")]
        public long CreatedTimestamp { get; set; }
        [JsonPropertyName("editedTimestamp")]
        public long EditedTimestamp { get; set; }
        [JsonPropertyName("isPrivate")]
        public bool IsPrivate { get; set; }
    }
}

File: NativeWrappers/Android/SimplySaveAndroid/app/src/main/java/com/simplysave/simplysaveandroid/dataTransfer/NoteData.kt
package com.simplysave.simplysaveandroid.dataTransfer

import kotlinx.serialization.Serializable

@Serializable
data class NoteData(
    val id: String,
    var title: String,
    var folderId: String,
    val createdTimestamp: Long,
    var editedTimestamp: Long,
    val isPrivate: Boolean,
)
